Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM geoffreybooth/meteor-base:2.5 AS meteor-builder
 
 ARG APPNAME
 
-# Make MATScommon discoverable
+RUN apt update && apt install -y ssh-client
```

Comment:
This line is no longer needed now that we're using jspdf 2.5. It can be removed.

Reply:
Done!